Ejercicio 1y2 topic 1

victorja opened the pull request:
Buscar en carpeta mismo nombre

victorja committed:
Ejercicio 1y2 topic 1

Agregado el fadeIn y el onclick

pabloascencao commented on topic1ej1y2/script.js:
```diff
@@ -0,0 +1,11 @@
+        $(document).ready(function() {
+        	$('#alert').on( "click", function(evt) {//ejercicio 2
+        		alert("I'm an alert");
+        		evt.stopImmediatePropagation();//evita que el onclick suceda 2 veces
```
Si encontras el porque sucede 2 veces vas a ver que no necesitas esto

pabloascencao commented on topic1ej1y2/style.css:
```diff
@@ -0,0 +1,83 @@
+	color: grey;
+}
+
+article{
```
hay otra forma de escribir las propiedades de esta clase?

pabloascencao commented on topic1ej1y2/style.css:
```diff
@@ -0,0 +1,83 @@
+}
+
+@media (max-width: 800px){
+article{
```
Idem

pabloascencao reviewed (changes requested):
Fijate el tema de la indentacion. es complejo para leer.

victorja committed:
Arreglado para evitar el stopimmediatepropagation()

Ver el js script

victorja committed:
actualizada funcion de aler msg

victorja committed:
arreglo css

puse text-align: center en todo el body y alinee a la izquierda los footers, elimine clases de mas al poner text-decoration: none a todos las tag a, antes lo hacia repetidas veces y eran muchos caracteres, lo mismo con el color de los tag a

pabloascencao commented:
Justamente el comentario de css que te deje no fue el que resolviste.

victorja committed:
cambio en padding

el cambio fue realizado en article y en el @media(max-width 800) article
